R/rd2qmd.R

```diff
@@ -19,14 +19,22 @@
   # first column (odd entries) of table in Arguments should not be wrapped
   idx <- grep("<td>", tmp)
   idx <- idx[seq_along(idx) %% 2 == 1]
-  tmp[idx] <- sub("<td>", '<td style = "white-space: nowrap; font-family: monospace; vertical-align: top">', tmp[idx])
+  tmp[idx] <- sub(
+    "<td>",
+    '<td style = "white-space: collapse; font-family: monospace; vertical-align: top">',
```

Comment:
This is the only change (`nowrap` -> `collapse`). The rest is just `air` formatting